components/automate-ui/src/app/pages/infra-proxy/chef-servers-details/chef-servers-details.component.html

```diff
@@ -0,0 +1,107 @@
+<app-client-runs-sidebar></app-client-runs-sidebar>
+
+<div class="container">
+  <main>
+    <chef-breadcrumbs>
+      <chef-breadcrumb [link]="['/infrastructure/chef-servers']">Chef Server</chef-breadcrumb>
+      Chef Server
+    </chef-breadcrumbs>
+    <chef-page-header>
+      <chef-heading>Chef Server</chef-heading>
+      <table>
+            <thead>
+              <tr class="detail-row">
+                <th class="id-column">FQDN</th>
+                <th class="id-column">IP Address</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr class="detail-row">
+                <td class="id-column">chef.io</td>
+                <td class="id-column">192.168.1.1</td>
+              </tr>
+            </tbody>
+          </table>
```

Comment:
we have a new angular component named `chef-table-new` that you might be able to use here.  It's under `app/components/table/table.component.ts`